functorch/_src/partitioners.py

```diff
@@ -91,13 +92,13 @@ def _extract_fwd_bwd_outputs(joint_module: fx.GraphModule):
     return fwd_outputs, bwd_outputs
 
 
-def _extract_fwd_bwd_modules(joint_module: fx.GraphModule, saved_values):
+def _extract_fwd_bwd_modules(joint_module: fx.GraphModule, saved_values, saved_sym_nodes=()):
```

Comment:
@ezyang @Chillee this is 2/2 where the saved sym nodes are separately handled, pruned out for unused ones, and then applied to the fwd/bwd args

as i write this, i wonder why i don't need to reverse the order of the contents of saved_sym_nodes as i place it at the beginning of bwd args.